config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -0,0 +1,27 @@
+periodics:
+- name: serviceaccount-admission-controller-migration
+  annotations:
+    testgrid-dashboards: sig-auth-gce
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  decorate: true
+  interval: 2h
```

Comment:
I'm ok with this to start until we get the job green, then we should probably lower it to ~1 time a day

Reply:
ack